inaugurator/server/idlistener.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, id, callback, channel):
         self._callback = callback
         self._notListeningAnymore = False
         self._channel.exchange_declare(
-            self._onExchangeDeclared, exchange=statusExchange(self._id), type='fanout')
```

Comment:
I suggest that you tighten the pika requirement; instead of >= 0.10.0, perhaps ==<new version>.
See the docker/build-inaugurator.dockerfile and inaugurator/server/pikapatchwakeupfromanotherthread.py.